packages/ffe-icons/README.md

```diff
@@ -95,55 +104,67 @@ Non-royal blue colors are left as-is, so details in other colors are left untouc
 
 In your markup:
 
-```
+```html
 <svg class="icon">
-    <use xmlns:xlink="http://www.w3.org/1999/xlink"
-         xlink:href="/app/symbol/ffe-icons.svg#person-ikon"></use>
+    <use
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        xlink:href="/app/symbol/ffe-icons.svg#person-ikon"
+    ></use>
     <!-- <desc>Alt text goes here</desc> -->
 </svg>
 ```
 
 Finally, depending on the current state of browsers and support requirements you may need to include the
 [SVG for Everybody](https://github.com/jonathantneal/svg4everybody) shim in your app.
 
-## Known issues
+## Adding a new icon
 
-See the current status on browser support for SVG fragments on [caniuse.com](caniuse.com/svg-fragment).
+So you've decided to contribute to our icons collection - that's great news! However, before you get started you should always consider if your icon would be useful to others outside your app/team. If you need help making that decision, or have any other questions - feel free to reach out to us either on Slack (#ext-designsystem) or via [GitHub Issues](https://github.com/SpareBank1/designsystem/issues).
+
+### Optimizing
 
-* At time of writing, the generated SVG sprite cannot be used as a source for `background-image` in your CSS like "single" SVGs can
+Many of the most popular visual SVG editors tends to generate code that is _less than ideal_ for the web. In addition to larger files with unnecessary code, things like CSS might leak into other parts of your applications, or some wonky property might make your bundling system freak out. In other words - you should familiarize with the SVG code, and make sure you optimize it properly.
 
+-   Remove unecessary code and properties. Tools like [this online SVG editor](https://petercollingridge.appspot.com/svg-editor) might be easier than manually using your editor.
 
-## Adding a new icon to ffe-icons
+-   Avoid using inline CSS. This might leak into other parts of your applications in addition to making it more complicated to control your from the outside. If you really have to use it - make sure the ID/classes you use are so unique that they wont interfer with any other styles.
 
-When your project produces a new icon, please add it to ffe-icons as a minor update to the current main version so other teams
-can make use of it. To do that, follow the steps described below. Ask around in the HipChat room Alliansens Frontend-forum
-if you get stuck.
+-   Avoid embedding the primary color, this should be controlled by CSS. Other colors that shouldn't be overridable might be emdedded in the file.
 
-Also before starting, verify your SVG meets the criteria listed in *icons/_ICON_REQUIREMENTS.md*
+-   The shape **must** fill the viewbox - minimum two edges of the square viewbox should be in contact with the shape. This is so margins can be fully controlled using CSS.
```

Comment:
Jeg vet at dette kravet ikke har vært populært blandt en del hos design fordi vi på noen ikoner har endt opp med å skalere viewbox'en slik at bredden på strekene har endret seg. Men det var noen der som skulle ta tak i det for et par år siden så inntill vi hører noe annet tenker jeg det er greit å ha det med for det er ikke lett å få en liste over ikoner til å se bra ut når ingen av dem er "like store" i den grad det gir mening å snakke om det for en SVG.

Reply:
Det høres fornuftig ut. Skal ærlig innrømme at akkurat dette punktet bare ble sakset direkte over. Jeg ser imidlertid problemet på begge sider. Kanskje det hadde vært bedre om vi fikk laget en mal med instruksjoner rettet mot designerene, fremfor at de som koder må "rydde opp" i filene etter de er overlevert?

Reply:
Jeg tror kanskje det er kommunisert ut og at jobben egentlig består i å fikse gamle ikoner som ble "ryddet opp" for flere år siden. Litt usikker på hva som er status der egentlig, tenkte bare å formidle hvorfor regelen er der sånn at flere husker det :)

Reply:
Ah skjønner. Da gjør jeg ikke noe mer her, men nyttig å vite om. Takk!